CONTRIBUTING.md

```diff
@@ -0,0 +1,143 @@
+
+#### Table Of Contents
+
+
+[Code of Conduct](#code-of-conduct)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Vulnerabilities](#reporting-vulnerabilities)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Pull Requests](#pull-requests)
+  * [Documentation Maintenance](#documentation-maintenance)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Google C++ Styleguide](https://google.github.io/styleguide/cppguide.html)
+  * [Documentation Styleguide](#documentation-styleguide)
+
+[Contribution Guide Thanks](#contribution-guide-thanks)
+
+## Code of Conduct
+
+1. Be nice and friendly
+2. Avoid any kind of abuses or strong language.
+3. Everything that You want to deliver to our community should be in well structured and polite form.
+4. Always make clear build before Pull Request
+5. Always reply on PR Code review questions and suggestions.
+6. Run all possible tests that You are able to run in Your environment before submitting PR.
+
+
+## How Can I Contribute?
+There are several ways to make contribution:
+* [Reporting Vulnerabilities](#reporting-vulnerabilities)
+* [Reporting Bugs](#reporting-bugs)
+* [Suggesting Enhancements](#suggesting-enhancements)
+* [Documentation Maintenance](#documentation-maintenance)
+
+
+### Reporting Vulnerabilities
+
+If You discovered the vulnerability:
+  1. DO NOT create an issue on GitHub, even with the pull request.
+     Help us avoid publicly known security issues.
+  2. Please, send us a detailed vulnerability report on the following e-mail:
```

Comment:
Seems something's missed.

Reply:
All e-mails will be added later, as soon as they will be created.